src/pyramid_marrowmailer/tests.py

```diff
@@ -88,14 +89,14 @@ def test_send(self):
         from pyramid_marrowmailer import get_mailer
         mailer = get_mailer(self.request)
 
-	message = mailer.new()
-	message.subject = "foobar"
-	message.to = "foobar@bar.com"
-	message.plain = "hi"
-	message.send()
-	self.assertEqual(self.handler.info, [])
+        message = mailer.new()
+        message.subject = "foobar"
+        message.to = "foobar@bar.com"
+        message.plain = "hi"
+        message.send()
+        self.assertEqual(self.handler.info, [])
+        # self.assertTrue('DELIVER' in self.handler.info[1])
```

Comment:
A year and a half has passed since I wrote the pull request, so I don't know anymore. 2 suggestions:

- Try to run the tests in Python 2.7 and Python 3.4 with the line uncommented so you find out why it is commented out (I really don't remember).
- Try to accept pull requests more quickly in the future.